makefile

```diff
@@ -1,5 +1,10 @@
 # start project configuration
 name := curator
+ifeq ($(shell go env GOOS), windows)
```

Comment:
we do this: in other makefiles: `ifeq ($(OS),Windows_NT)` which doesn't require a fork/exec on every make invocation

Reply:
Changed this to use `${GOOS}` environment variable.